<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0.0000" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2024-09-05T20:29:11" tests="0" file="cypress/integration/plugins/index-management-dashboards-plugin/reindex_spec.js" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Reindex" timestamp="2024-09-05T20:29:11" tests="0" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Reindex validation error" timestamp="2024-09-05T20:29:11" tests="1" time="0.0000" failures="1">
    <testcase name="Reindex Reindex validation error &quot;before each&quot; hook for &quot;source validation failed&quot;" time="0.0000" classname="&quot;before each&quot; hook for &quot;source validation failed&quot;">
      <failure message="Timed out retrying after 60000ms: Expected to find content: &apos;Rows per page&apos; but never did.

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Reindex`" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 60000ms: Expected to find content: 'Rows per page' but never did.

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Reindex`
    at Context.eval (http://localhost:5601/__cypress/tests?p=cypress/integration/plugins/index-management-dashboards-plugin/reindex_spec.js:171:8)]]></failure>
    </testcase>
  </testsuite>
</testsuites>